components/list/demo/list-item-sample.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta http-equiv="X-UA-Compatible" content="ie=edge">
+	<meta charset="UTF-8">
+	<link rel="stylesheet" href="../../demo/styles.css" type="text/css">
+	<script src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+	<script type="module">
+		import '../../demo/demo-page.js';
+		import '../list-item-content.js';
+		import './list-item-sample.js';
+	</script>
+</head>
+
+<body unresolved>
+
+	<d2l-demo-page page-title="d2l-list-item-sample">
+
+		<h2>List item sample for development</h2>
```

Comment:
To be semantically correct, the items should be placed inside of a `d2l-list` element, even for the demo.  This will allow a tester to check both the behavior of `d2l-list` with custom items as well as the behavior of the mixin.

Can we just add this demo to the existing demo page?  So far I think this could go quite nicely with the rest of the demos snippets, but we should just provide demo for the bits that are different or could be affected by overrides, and omitting what's already covered by the standard list-item demo.
```
<d2l-list>
    <d2l-list-item-sample key="foobar selectable="true" draggable="true">
       ...
    </d2l-list-item-sample>
</d2l-list>
```

Another minor note: the string values should be omitted from the boolean attributes:
```
<d2l-list-item-sample key="foobar selectable draggable disabled>...</d2l-list-item-sample>
```
The exception to that rule is aria attributes, which awkwardly expect/require string values.

Reply:
I see you comment in Slack now.  Hopefully, unless we need a special page, the mixin functionality can eventually be adequately demo'd on the existing page.

Reply:
Yeah, the sample is only for development of the mixins in the interim before we actually change the list-item itself. It will be removed and demoed properly on the existing demo page.